pkg/api/types.go

```diff
@@ -337,6 +354,13 @@ const (
 	// used for PersistentVolumes where the bound PersistentVolumeClaim was deleted
 	// released volumes must be recycled before becoming available again
 	VolumeReleased PersistentVolumePhase = "Released"
+	// used for PersistentVolumes that have been recycled and can be made available
+	// again to new PersistentVolumeClaims
+	VolumeRecycled PersistentVolumePhase = "Recycled"
```

Comment:
So you're using this to signal another component (the binder?) that this PV can be used?  Does the binder clear this phase and put it into Available?  How is this different from Available?